odxtools/multiplexer.py

```diff
@@ -168,8 +166,8 @@ class Multiplexer(DopBase):
     long_name: str
     byte_position: int
     switch_key: MultiplexerSwitchKey
-    default_case: Optional[MultiplexerDefaultCase] = None
-    cases: Optional[List[MultiplexerCase]] = None
+    default_case: Optional[MultiplexerDefaultCase]
+    cases: Optional[List[MultiplexerCase]]
```

Comment:
```suggestion
    cases: List[MultiplexerCase]
```

Reply:
Good suggestion. I've already implemented these changes but I'd like to keep them out of this PR due to their off-topic nature. (will do a separate PR once this one is merged to avoid conflicts and confusion.)